spec/controllers/admin/cms/collections_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Admin::CollectionsController, type: :controller do
```

Comment:
to your question in flowdock about FactoryGirl...might i suggest using RSpec doubles rather than FactoryGirl-generated model instances in your controller specs? Using doubles mean that your controller specs will _only_ test your controller (a good thing) rather than any model behaviour (that's for the model specs)

good rule of thumb: a class' specs should only execute that class. use doubles to test method calls to other classes, and mocked responses